base/iterators.jl

```diff
@@ -435,9 +424,9 @@ julia> collect(Iterators.rest([1,2,3,4], 2))
 ```
 """
 rest(itr,state) = Rest(itr,state)
+rest(itr) = itr
```

Comment:
This is a bit confusing since often the functions `first` and `rest` are synonyms for `head` and `tail`, i.e. `rest(x)` drops the first element.

Reply:
I'm happy to drop this method, though it is useful (see the use in _totuple).